x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/SqlClearCursorRequest.java

```diff
@@ -43,9 +48,11 @@
         PARSER.declareString(optionalConstructorArg(), MODE);
         PARSER.declareString(optionalConstructorArg(), CLIENT_ID);
         PARSER.declareString(optionalConstructorArg(), VERSION);
+        PARSER.declareBoolean(SqlClearCursorRequest::binaryCommunication, BINARY_FORMAT);
     }
 
     private String cursor;
+    private Boolean binaryCommunication = BINARY_COMMUNICATION;
```

Comment:
Sorry, another question: why `Boolean` and not `boolean`?

Reply:
Same as for `SqlQueryRequest`, I'm using the default value (ie. `CoreProtocol.BINARY_COMMUNICATION`) that is Boolean and today is NULL.